src/plasoscaffolder/bll/mappings/mapping_helper.py

```diff
@@ -15,18 +14,21 @@ def __init__(self, template_path: str):
       template_path (str): the path to the template directory
     """
     super().__init__()
-    self.template_path = template_path
+    self.__template_path = template_path
 
-  def _get_template_environment(self) -> Environment:
+  @property
```

Comment:
If you use @property make it look and feel like a instance attribute

```
@property
def template_environment(self):
  ...
```

Otherwise just remove `@property`